source/2019-12-20-the-ember-times-issue-129.md

```diff
@@ -0,0 +1,159 @@
+---
+title: The Ember Times - Issue No. 129
+author: Chris Ng, Amy Lam, Isaac Lee, Godfrey Chan, Jessica Jordan
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
+alias : "blog/2019/12/20-the-ember-times-issue-129.html"
+responsive: true
+---
+
+Happy Holidays, Emberistas! 🐹🌲⛄️
+
+Ember Octane is here 🐹🎉,
+new ember-autofocus-modifier 💡,
+3 new RFCs for template helpers 3️⃣,
+ember-apollo-client v2 released 🎉
+and join us for the last 11 days of DecEmber 1️⃣️⃣1️⃣!
+
+READMORE
+
+---
+
+## [Octane is Here 🎉](https://blog.emberjs.com/2019/12/20/octane-is-here.html)
+
+[Ember 3.15](https://blog.emberjs.com/2019/12/20/ember-3-15-released.html) was released this week. In addition to the usual new features, improvements and bug fixes, this release also marks the official release of Ember's first edition - **Ember Octane**!
+
+<!--alex ignore easy-->
+But what is Ember Octane? Ember Octane is the new, recommended way for developers to build Ember applications and addons.
+Octane features a new reactivity model, a renewed practice of building components and a stronger focus on HTML driven web development than ever before. Octane aims to make building Ember apps for developers not only more easy, but also **more fun** right from the start.
+And on top of that, Ember Octane is **fully opt-in** and **interoperable** for existing Ember apps - which means you can try out the new way of building modern Ember Octane apps gradually, without having to rewrite your entire app today.
+
+Want to know all about Ember Octane? Learn more about what's new in Octane and how to get started [in the official Ember Octane release announcement](https://blog.emberjs.com/2019/12/20/octane-is-here.html) by [Yehuda Katz (@wycats)](https://github.com/wycats)! Need help? Check out **#topic-octane-migration** channel on the [Ember Discord](https://discordapp.com/invite/emberjs)!
```

Comment:
```suggestion
Want to know all about Ember Octane? Learn more about what's new in Octane and how to get started [in the official Ember Octane release announcement](https://blog.emberjs.com/2019/12/20/octane-is-here.html) by [Yehuda Katz (@wycats)](https://github.com/wycats)! Need help? Check out the **#topic-octane-migration** channel on the [Ember Discord](https://discordapp.com/invite/emberjs)!
```

Reply:
My bad, I missed a "the" here

Reply:
Good catch, I updated it in a separate commit :+1: